vxquery-xtest/src/main/java/org/apache/vxquery/xtest/TestRunner.java

```diff
@@ -181,11 +181,13 @@ public TestCaseResult run(final TestCase testCase) {
             res.time = end - start;
         }
         if (opts.showResult) {
```

Comment:
Yes it is. But, there is a bug. opts.showResult should instead be opts.showresult. So it will fail. Do you want me to push it again, or will you modify it?